integration/test/organization/main_test.go

```diff
@@ -0,0 +1,137 @@
+// +build k8srequired
+
+package organization
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	corev1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/core/v1alpha1"
+	providerv1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/provider/v1alpha1"
```

Comment:
```suggestion
	corev1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/core/v1alpha1"
	providerv1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
```

Can we use v3 for all of these?

Reply:
Fixed